x-pack/plugins/security_solution/package.json

```diff
@@ -25,7 +25,8 @@
     "mappings:generate": "node scripts/mappings/mappings_generator",
     "mappings:load": "node scripts/mappings/mappings_loader",
     "junit:transform": "node scripts/junit_transformer --pathPattern '../../../target/kibana-security-solution/cypress/results/*.xml' --rootDirectory ../../../ --reportName 'Security Solution Cypress' --writeInPlace",
+    "openapi:lint": "npx @redocly/cli lint --config ../../../packages/kbn-openapi-generator/redocly_linter/config.yaml $(find common/api -name '*.schema.yaml')",
```

Comment:
I'm not sure if we run any other commands like this, through `npx` without having it installed locally. 
@elastic/kibana-operations - what do you think, are `npx` commands OK, or should we ask for the library to be installed as a dev dependency for it to be used?

Reply:
Addressed here https://github.com/elastic/kibana/pull/171851#discussion_r1406360228